README.md

```diff
@@ -190,11 +190,13 @@ Check out the [$care-gaps operation spec](https://build.fhir.org/ig/HL7/davinci-
 
 This operation retrieves all the data requirements for a given measure as a FHIR library.
 
-Required parameters include:
+Optional parameters for this function include:
 
 - `periodStart`: start of the measurement period
 - `periodEnd`: end of the measurement period
 
+If `periodStart` and `periodEnd` params are omitted, the measurement period for the operation will default to the `effectivePeriod` of the referenced FHIR Measure. If no `effectivePeriod` property is present, `dateFilters` will be excluded from the returned FHIR Library entirely.
```

Comment:
Do we want to add a description of the logic for when only a periodStart or periodEnd is added? Or is that unnecessary